cnf/ext/osgi_repos.bnd

```diff
@@ -0,0 +1,55 @@
+# Configure Repositories
+-plugin.1.R7.API: \
+	aQute.bnd.repository.maven.pom.provider.BndPomRepository; \
+		snapshotUrls=https://oss.sonatype.org/content/repositories/osgi/; \
+		releaseUrls=https://repo.maven.apache.org/maven2/; \
+		revision=org.osgi.enroute:osgi-api:7.0.0; \
```

Comment:
I guess I am not sure of the ongoing status of enRoute with the shutdown of the OSGi Alliance and the movement of OSGi specification development to Eclipse. I think Bndtools should consider moving away from enRoute artifacts and use OSGi artifacts.

Reply:
I guess we can clarify this in the upcoming weeks in our steering Committee calls and I will change it accordingly.